src/locale/misc/smartSearch/en.yml

```diff
@@ -131,6 +132,26 @@ person_tags:
     examples:
         one: "Add people with at least one of the following tags: 'Member', 'Activist'"
         two: "Remove people with all of the following tags: 'Board member'"
+task:
+    inputString: "{addRemoveSelect} people who have {taskStatusSelect} {taskSelect}{campaignSelect} {matchingSelect} {timeFrame}"
+    addRemoveSelect:
+        add: Add
+        sub: Remove
+    examples:
+        one: Add people who have completed task "Tell your friends" at least once at any point in time
+        two: Add people who have ignored any task in any campaign between 2 and 5 times before today 
+    taskStatusSelect:
+        completed: completed
+        ignored: ignored
+        assigned: been assigned to
```

Comment:
I think this should just be "been assigned", not "been assigned to", but let's consult a native English speaker like @djbusstop or @lowlandjuju. 😊 

Should it be "people who have been assigned any task…" or "people who have been assigned to any task…"?

![image](https://user-images.githubusercontent.com/550212/151250206-fd6e14f4-4c1f-47ff-8f2a-06ec790d6b36.png)

Reply:
I don't think it really matters either way, but I guess the best one is "have been assigned" without the word "to"

Reply:
Alright I updated to "been assigned". The shorter the better!